x/marketmaker/client/cli/tx.go

```diff
@@ -0,0 +1,218 @@
+package cli
+
+// DONTCOVER
+// client is excluded from test coverage in MVP version
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/version"
+	"github.com/cosmos/cosmos-sdk/x/gov/client/cli"
+	gov "github.com/cosmos/cosmos-sdk/x/gov/types"
+
+	"github.com/cosmosquad-labs/squad/v2/x/marketmaker/types"
+)
+
+// GetTxCmd returns a root CLI command handler for all x/marketmaker transaction commands.
+func GetTxCmd() *cobra.Command {
+	marketmakerTxCmd := &cobra.Command{
+		Use:                        types.ModuleName,
+		Short:                      "MarketMaker transaction subcommands",
+		DisableFlagParsing:         true,
+		SuggestionsMinimumDistance: 2,
+		RunE:                       client.ValidateCmd,
+	}
+
+	marketmakerTxCmd.AddCommand(
+		NewApplyMarketMaker(),
+		NewClaimIncentives(),
+	)
+
+	return marketmakerTxCmd
+}
+
+// NewApplyMarketMaker implements apply market maker command handler.
+func NewApplyMarketMaker() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "apply [pool-ids]",
+		Args:  cobra.MaximumNArgs(1),
+		Short: "Apply to be a market maker",
+		Long: strings.TrimSpace(
+			fmt.Sprintf(`
+Apply to be a market maker for a number of pairs. The deposit amount defined in params is required to deposit, and the amount is expected to be refunded when you are either included or rejected by the community (through a governance proposal).
+
+Example:
+$ %s tx %s apply 1 --from mykey
+$ %s tx %s apply 1,2 --from mykey
+`,
+				version.AppName, types.ModuleName,
+				version.AppName, types.ModuleName,
+			),
+		),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			clientCtx, err := client.GetClientTxContext(cmd)
+			if err != nil {
+				return err
+			}
+
+			farmer := clientCtx.GetFromAddress()
+			pairIds := []uint64{}
+			switch len(args) {
+			case 1:
+				pairIdsStr := strings.Split(args[0], ",")
+
+				for _, i := range pairIdsStr {
+					pairId, err := strconv.ParseUint(i, 10, 64)
+					if err != nil {
+						return fmt.Errorf("parse pair id: %w", err)
+					}
+					pairIds = append(pairIds, pairId)
+				}
+			default:
+				return fmt.Errorf("only pool-ids flag must be specified")
+			}
```

Comment:
How about passing `cobra.ExactArgs(1)` for `Args` field instead of checking the length of `args`?